salt/metalk8s/addons/dex/deployed/service-configuration.sls

```diff
@@ -0,0 +1,66 @@
+include:
+  - .namespace
+
+{%- set dex_config = salt.metalk8s_kubernetes.get_object(
+        kind='ConfigMap',
+        apiVersion='v1',
+        namespace='metalk8s-auth',
+        name='metalk8s-dex-config'
+  )
+%}
+
+{%- if dex_config is none %}
+
+Create dex-config ConfigMap:
+  metalk8s_kubernetes.object_present:
+    - manifest:
+        apiVersion: v1
+        kind: ConfigMap
+        metadata:
+          name: metalk8s-dex-config
+          namespace: metalk8s-auth
+        data:
+          config.yaml: |-
+            apiVersion: addons.metalk8s.scality.com
+            kind: DexConfig
+            spec:
+              deployment:
+                replicas: 2
+              localuserstore:
+                enabled: true
+                email: "admin@metalk8s.invalid"
+                hash: "$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W"
+              connectors:
+                enabled: false
+                type: ldap
+                name: LDAP
+                id: ldap
+                config:
+                  host: ldap.metalk8s.invalid:636
+                  startTLS: true
+                  rootCAData: "{{ salt['hashutil.base64_encodefile']('/etc/metalk8s/pki/dex/ca.crt') | replace('\n', '') }}"
```

Comment:
Why would the Dex CA be the root CA of the LDAP directory server?

Reply:
This should not be the case, i only found out that without a value in this field, the Dex containers crashed.

Reply:
Then let's use... 'some' other value, not an actually used cert.

Reply:
IMO just not prefill `connectors` and just have an empty list by default with documentation explaning how to fill it